tutorials/fundamentals/subvsslice.md

```diff
@@ -0,0 +1,11 @@
+JavaScript's string `substring()` and `slice()` functions behave similarly but have a few key differences.
+With `substring()`, the two arguments are swapped if the first argument is greater than the second argument, and
+negative and `NaN` values are treated as 0. With `slice()`, you can enter a negative value as an argument and
```

Comment:
I'd make a separate paragraph explaining the `NaN` difference and the "swapped if first argument is greater" example. The first paragraph should highlight the fact that `slice()` supports negative values, with an example.

So 3 paragraphs, with an example after each one:

1) Negative values with `slice()` vs `substring()`

2) 1st arg is greater than 2nd arg, with `substring()` vs `slice()`

3) How they handle `NaN`.